core/app/models/spree/order_updater.rb

```diff
@@ -73,14 +73,30 @@ def update_order_total
 
     def update_adjustment_total
       recalculate_adjustments
-      order.adjustment_total = line_items.sum(:adjustment_total) +
-                               shipments.sum(:adjustment_total)  +
+      lis = line_items.pluck(:adjustment_total, :included_tax_total, :additional_tax_total, :promo_total).reduce([0,0,0,0]) { |accum, item|
+        accum[0] += item[0]
+        accum[1] += item[1]
+        accum[2] += item[2]
+        accum[3] += item[3]
+        accum
+      }
+
+      shis = shipments.pluck(:adjustment_total, :included_tax_total, :additional_tax_total, :promo_total).reduce([0,0,0,0]) { |accum, item|
+        accum[0] += item[0]
+        accum[1] += item[1]
+        accum[2] += item[2]
+        accum[3] += item[3]
+        accum
+      }
+
+      order.adjustment_total = lis[0] +
+                               shis[0]  +
```

Comment:
Unnecessary spacing detected.<br>Use 2 (not 25) spaces for indenting an expression spanning multiple lines.